torch/_dynamo/compiled_autograd.py

```diff
@@ -71,21 +71,35 @@ def maybe_clone(x):
 # function is called. It's possible to avoid lazy binding and instead bind
 # all of this upfront (perhaps at import time) via codegen changes.
 class OpNamespace:
-    def add(self, name, fn):
-        assert not hasattr(self, name)
-        result = Op(name, fn)
+    def __init__(self):
+        self.custom_function_name_counter = {}
+
+    def add(self, name, fn, is_custom_function=False):
+        if is_custom_function:
+            # TODO(rzou): wut
+            name = "f" + name
```

Comment:
name can start with numbers, so the f is to give a better name... TODO on how to get a better name out of this